src/engines/html5/html5.js

```diff
@@ -33,20 +33,24 @@ export default class Html5 extends FakeEventTarget implements IEngine {
    * @private
```

Comment:
should be `@type {?IMediaSourceAdapter}`

Reply:
fixed